src/tags/index.js

```diff
@@ -129,23 +129,68 @@ module.exports = function(fractal){
                 }
             }
         },
+
         trans(Twig) {
             return {
                 type: 'trans',
-                regex: /^trans/,
+                regex: /^trans(.+)?/,
                 next: ['endtrans'],
                 open: true,
                 compile: function (token) {
                     return token;
                 },
                 parse: function (token, context, chain) {
-                    return {
-                        chain: chain,
-                        output: Twig.parse.apply(this, [token.output, context])
-                    };
+                    var that = this;
+                    return Twig.parseAsync.call(that, token.output, context)
+                    .then(function (value) {
```

Comment:
Indentation error here – quickest fix would be to join the `.then()` onto the previous line.

Reply:
I don't like it too, but as mention, the idea is to keep the style of the `twigjs` because this code is basically extending it's one. All the tag definitions are using this syntax: https://github.com/twigjs/twig.js/blob/master/src/twig.logic.js#L95